### PR TITLE
Add script for opening Github PR

### PR DESCRIPTION
**Pull request by Allda**
The script opens a pull request in a github. The script will be used in
CI pipeline.

JIRA: ISV-860

### CODE REVIEW

File: operatorcert/entrypoints/github_pr.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    """
+    Create a pull request body
+
+    Args:
+        args (Any): CLI arguments
+
+    Returns:
+        str: Pull request text body
+    """
+    title_parts = args.title.split()
```
Comment (wcheang) [inline operatorcert/entrypoints/github_pr.py:131]: This looks like something that's done more safely in `parse_pr_title`

File: operatorcert/entrypoints/github_pr.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    """
+    Setup argument parser
+
+    Returns:
+        Any: Initialized argument parser
+    """
+    parser = argparse.ArgumentParser(description="Github PR cli tool.")
+
+    parser.add_argument(
+        "--repository-url",
```
[the remark this answers is not in the document]
Comment (Allda) [reply operatorcert/entrypoints/github_pr.py]: Fixed

File: operatorcert/entrypoints/github_pr.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        repository_url (str): Git repository URL
+        branch (str): Git branch
+
+    Returns:
+        str: Git head for PR that serves as a source
+    """
+    parsed_url = giturlparse.parse(repository_url)
+    path = parsed_url.owner
+
+    namespace = path.split("/")[0]
```
[the remark this answers is not in the document]
Comment (Allda) [reply operatorcert/entrypoints/github_pr.py]: Nice catch! Fixed

File: tests/entrypoints/test_github_pr.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+def test_get_pr_body() -> None:
+    args = MagicMock()
+    args.title = "operator foo (1.0.0)"
+    args.cert_project_id = "0123"
+    args.test_result_url = "https://foo.com/tests"
+    args.test_logs_url = "https://foo.com/logs"
+
+    resp = github_pr.get_pr_body(args)
+
+    expected = """**New operator bundle**
```
[the remark this answers is not in the document]
Comment (Allda) [reply tests/entrypoints/test_github_pr.py]: I learn new stuff every day! thanks

File: operatorcert/entrypoints/github_pr.py (hunk 1 above)
Comment (Allda) [reply operatorcert/entrypoints/github_pr.py]: Fixed

**Review by wcheang — APPROVED** [review]
LGTM :+1:

**Pull request merged**